src/sentry/workflow_engine/handlers/condition/event_frequency_base_handler.py

```diff
@@ -30,16 +30,13 @@ class _QSTypedDict(TypedDict):
 
 
 class BaseEventFrequencyConditionHandler(ABC):
-    @property
-    @abstractmethod
-    def base_handler(self) -> Self:
+    intervals: ClassVar[dict[str, tuple[str, timedelta]]] = STANDARD_INTERVALS
+
+    @classmethod
+    def base_handler(cls) -> type["BaseEventFrequencyConditionHandler"]:
```

Comment:
having some trouble not having this be a `classmethod`. if i make it a `ClassVar`, i get `TypeError: Can't instantiate abstract class BaseEventFrequencyConditionHandler` when calling `handler().get_rate_bulk`